tests/manager/manager-executeWriteCommand_error-003.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for unsupported update operator
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$command = new MongoDB\Driver\Command([
+    'findAndModify' => COLLECTION_NAME,
+    'query' => ['_id' => 'foo'],
+    'upsert' => true,
+    'new' => true,
+]);
+
+try {
+    $manager->executeWriteCommand(DATABASE_NAME, $command);
+} catch (MongoDB\Driver\Exception\CommandException $e) {
+	printf("CommandException: %s\n", $e->getMessage());
+
+    echo "\n===> ResultDocument\n";
```

Comment:
I think we can remove this. Simply `var_dump()`-ing the property after the `printf()` above should be fine, as that's all we're doing in this test.

The tests that use `printWriteResult()` are some of the oldest, so they often have funny formatting and makeshift titles for output.